CIP-0187? | Utilization-Scaled Pledge Bonus (#1193)
cardano-foundation/CIPs: CIP-XXXX/utilization_scaled_pledge_bonus.md

A coordinated delegator like Pool Ranger — which performs the rational-delegator function the protocol assumes exists — feels this defect most acutely. To maximize member ROA it must concentrate stake into a high-pledge pool. To gain bargaining leverage over the SPO it must concentrate further still. Both moves dilute. The protocol gives nothing in return and the members pay the cost.

A direct fix — lowering `a₀` (the pledge influence factor) — is blocked: `a₀` is doing important Sybil-resistance work and cannot be removed without replacing the mechanism it carries.

Is there a way to remove the dilution directly, by reshaping the existing pledge-bonus formula so that it scales with pool stake — without inventing new parameters, new on-chain state, or special treatment for any class of delegator? A change that helps SPOs, helps every delegator, helps cooperatives like Pool Ranger, supports decentralization, and preserves protocol health?

## Glossary

> This document is intended to stand alone. The tables below define every symbol and term used in the rest of the document. Readers familiar with Cardano staking can skim; readers new to it can return here whenever a term is unclear.

Reviewer: ```suggestion
## Glossary

The tables below define every symbol and term used in the CIP. Readers familiar with Cardano staking can skim; readers new to it can return here whenever a term is unclear.
```
It has been hard to find any part of a CIP that can "stand alone" unless it is _creating_ that resource.  In the case of background material for a CIP already defined elsewhere, it doesn't really have to stand _anywhere_ 😅

However since you would already be creating an _Appendix_ I can see an advantage to creating another one as a _Glossary_ (or having an "optional" Glossary main heading: I will argue that this is permissible) because of the advantage to completely disambiguating the mathematical terms you are using.

Of course this means we would have to consider all these definitions as part of the CIP material under review... and therefore could experience some delays because all 3 editors would have double-check them for accuracy and possible bias.
* This is one reason why it might be better to find a link to such a canonical glossary of terms relevant to reward sharing & PoS incentives in Cardano (maybe @Cerkoryn would know of one?).

Reviewer: Resolved as @johnshearing explains in https://github.com/cardano-foundation/CIPs/pull/1193#issuecomment-4478130821.
